### ISSUE TITLE
Typo in Cisco decoder

### ISSUE THREAD

**Issue by juliamagan** [wazuh/wazuh#22429]
|Wazuh version| Component | Action type |
|---| --- | --- |
| 4.7.2-40711 | Decoders |Error |

## Description

An error has been found in the prematch regex of a Cisco parent decoder, in `ruleset/decoders/0065-cisco-ios_decoders.xml`

https://github.com/wazuh/wazuh/blob/c6412e52263bdce7b8be0e4625466b63b43aa65e/ruleset/decoders/0065-cisco-ios_decoders.xml#L116

The regex part `\d\p**` is not allowing to match the parent and get information from the children, forcing the user to create a custom one in `/var/ossec/etc/decoder` or modify the default one.

### Current results

```
root@ubuntu-manager:/home/vagrant# /var/ossec/bin/wazuh-logtest
Starting wazuh-logtest v4.7.2
Type one log per line

cisco-ios Mar 7 03:48:28 XXXXX 785839: 785846: Mar 7 2024 03:48:27.622 UTC: %DOT1X-5-FAIL: Switch 1 R0/0: sessmgrd: Authentication failed for client (9088.55af.8810) with reason (No Response from Client) on Interface Fi1/0/19 AuditSessionID C204280A000117CDA045ADDF


**Phase 1: Completed pre-decoding.
	full event: 'cisco-ios Mar 7 03:48:28 XXXXX 785839: 785846: Mar 7 2024 03:48:27.622 UTC: %DOT1X-5-FAIL: Switch 1 R0/0: sessmgrd: Authentication failed for client (9088.55af.8810) with reason (No Response from Client) on Interface Fi1/0/19 AuditSessionID C204280A000117CDA045ADDF'

**Phase 2: Completed decoding.
	No decoder matched.

**Phase 3: Completed filtering (rules).
	id: '2501'
	level: '5'
	description: 'syslog: User authentication failure.'
	groups: '['syslog', 'access_control', 'authentication_failed']'
	firedtimes: '1'
	gdpr: '['IV_35.7.d', 'IV_32.2']'
	gpg13: '['7.8']'
	hipaa: '['164.312.b']'
	mail: 'False'
	nist_800_53: '['AU.14', 'AC.7']'
	pci_dss: '['10.2.4', '10.2.5']'
	tsc: '['CC6.1', 'CC6.8', 'CC7.2', 'CC7.3']'
**Alert to be generated.

```

### Expected results

```root@ubuntu-manager:/home/vagrant# /var/ossec/bin/wazuh-logtest
Starting wazuh-logtest v4.7.2
Type one log per line

cisco-ios Mar 7 03:48:28 XXXXX 785839: 785846: Mar 7 2024 03:48:27.622 UTC: %DOT1X-5-FAIL: Switch 1 R0/0: sessmgrd: Authentication failed for client (9088.55af.8810) with reason (No Response from Client) on Interface Fi1/0/19 AuditSessionID C204280A000117CDA045ADDF

**Phase 1: Completed pre-decoding.
	full event: 'cisco-ios Mar 7 03:48:28 XXXXX 785839: 785846: Mar 7 2024 03:48:27.622 UTC: %DOT1X-5-FAIL: Switch 1 R0/0: sessmgrd: Authentication failed for client (9088.55af.8810) with reason (No Response from Client) on Interface Fi1/0/19 AuditSessionID C204280A000117CDA045ADDF'

**Phase 2: Completed decoding.
	name: 'cisco-ios'
	cisco.facility: 'DOT1X'
	cisco.mnemonic: 'FAIL'
	cisco.severity: '5'

**Phase 3: Completed filtering (rules).
	id: '4715'
	level: '0'
	description: 'Cisco IOS notification message - FAIL'
	groups: '['syslog', 'cisco_ios']'
	firedtimes: '1'
	mail: 'False'
```
### Log examples
```
cisco-ios Mar 7 03:48:28 XXXXX 785839: 785846: Mar 7 2024 03:48:27.622 UTC: %DOT1X-5-FAIL: Switch 1 R0/0: sessmgrd: Authentication failed for client (9088.55af.8810) with reason (No Response from Client) on Interface Fi1/0/19 AuditSessionID C204280A000117CDA045ADDF
```

**Comment by jk-olaoluwa** [comment]
### Review

### Testing the old decoder rule on Wazuh Manager v4.4.5
``` console
[root@wazuh-server wazuh-user]# /var/ossec/bin/wazuh-logtest
Starting wazuh-logtest v4.4.5
Type one log per line

cisco-ios Mar 7 03:48:28 XXXXX 785839: 785846: Mar 7 2024 03:48:27.622 UTC: %DOT1X-5-FAIL: Switch 1 R0/0: sessmgrd: Authentication failed for client (9088.55af.8810) with reason (No Response from Client) on Interface Fi1/0/19 AuditSessionID C204280A000117CDA045ADDF

** Wazuh-Logtest: WARNING: (7612): Rule ID '100200' is duplicated. Only the first occurrence will be considered.

**Phase 1: Completed pre-decoding.
        full event: 'cisco-ios Mar 7 03:48:28 XXXXX 785839: 785846: Mar 7 2024 03:48:27.622 UTC: %DOT1X-5-FAIL: Switch 1 R0/0: sessmgrd: Authentication failed for client (9088.55af.8810) with reason (No Response from Client) on Interface Fi1/0/19 AuditSessionID C204280A000117CDA045ADDF'

**Phase 2: Completed decoding.
        No decoder matched.

**Phase 3: Completed filtering (rules).
        id: '2501'
        level: '5'
        description: 'syslog: User authentication failure.'
        groups: '['syslog', 'access_control', 'authentication_failed']'
        firedtimes: '1'
        gdpr: '['IV_35.7.d', 'IV_32.2']'
        gpg13: '['7.8']'
        hipaa: '['164.312.b']'
        mail: 'False'
        nist_800_53: '['AU.14', 'AC.7']'
        pci_dss: '['10.2.4', '10.2.5']'
        tsc: '['CC6.1', 'CC6.8', 'CC7.2', 'CC7.3']'
**Alert to be generated.
```

### Testing the new decoder rule
``` console
[root@wazuh-server wazuh-user]# /var/ossec/bin/wazuh-logtest
Starting wazuh-logtest v4.4.5
Type one log per line

cisco-ios Mar 7 03:48:28 XXXXX 785839: 785846: Mar 7 2024 03:48:27.622 UTC: %DOT1X-5-FAIL: Switch 1 R0/0: sessmgrd: Authentication failed for client (9088.55af.8810) with reason (No Response from Client) on Interface Fi1/0/19 AuditSessionID C204280A000117CDA045ADDF

** Wazuh-Logtest: WARNING: (7612): Rule ID '100200' is duplicated. Only the first occurrence will be considered.

**Phase 1: Completed pre-decoding.
        full event: 'cisco-ios Mar 7 03:48:28 XXXXX 785839: 785846: Mar 7 2024 03:48:27.622 UTC: %DOT1X-5-FAIL: Switch 1 R0/0: sessmgrd: Authentication failed for client (9088.55af.8810) with reason (No Response from Client) on Interface Fi1/0/19 AuditSessionID C204280A000117CDA045ADDF'

**Phase 2: Completed decoding.
        name: 'cisco-ios'
        cisco.facility: 'DOT1X'
        cisco.mnemonic: 'FAIL'
        cisco.severity: '5'

**Phase 3: Completed filtering (rules).
        id: '4715'
        level: '0'
        description: 'Cisco IOS notification message - FAIL'
        groups: '['syslog', 'cisco_ios']'
        firedtimes: '1'
        mail: 'False'
```

### Conclusion
The modified decoder rule works as intended on Wazuh v4.4.5

**Comment by ooniagbi** [comment]
https://github.com/wazuh/wazuh/pull/22620 merged to 4.8.2.
